### PR TITLE
Added Cpu stub class

### LINKED ISSUE #21: Create Cpu class

### PR DESCRIPTION
**Pull request by jazzybear1**
Hey guys, 

Please check out these changes when you get time 👾 I added the Cpu stub class and fixed a typo in the Readme. Thanks!

### CODE REVIEW

**Review by turtleboy00 — APPROVED** [review]
Looks good to me! Good job 👏

File: inc/Cpu.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class Cpu {
+public:
+  Cpu() = default;
+  ~Cpu() = default;
+
+  void* GetRegister(std::string);
+  void RunInstruction(std::string);
+  void SetRegister(std::string, void*);
+
+private:
```
Comment (richardllara05) [inline inc/Cpu.h:16]: Check the UML for new changes.

Comment (richardllara05) [inline inc/Cpu.h:16]: Check the UML for new changes.

**Review by jrbetancourt98 — APPROVED** [review]

File: inc/Cpu.h (hunk 1 above)
Comment (jaburialeh) [reply inc/Cpu.h:18]: I added the new Opcode member from the UML. Thanks

**Pull request merged**